root/edit/search_macros.tt

```diff
@@ -299,6 +301,23 @@
   </select>
 [% END %]
 
+[% MACRO predicate_applied_edits(field, field_contents) WRAPPER wfield predicate='id' %]
```

Comment:
Is there any reason we can't use `predicate_int` instead? Looks about the same but I might have missed something.